src/__init__.py

```diff
@@ -19282,6 +19299,8 @@ def __init__(self, out, clips, method):
         self.ctm = mupdf.FzMatrix()
         self.rot = mupdf.FzMatrix()
         self.lastpoint = mupdf.FzPoint()
+        self.firstpoint = mupdf.FzPoint()
+        self.havemove = 0
```

Comment:
Minor suggestion - we could set self.firstpoint to None here, and then use `if self.firstpoint is not None: ...` instead of `if self.havemove: ...` later on.

Reply:
how to reflect that in extract.i?